wisp-tracing/src/main/kotlin/wisp/tracing/Tracing.kt

```diff
@@ -92,6 +99,7 @@ inline fun <T: Any?> Tracer.scoped(
 /**
  * Creates a span called [name] which is a child of [parent].
  */
+@Deprecated("Don't create new spans this way, instead call Tracer.trace or Tracer.traceWithSpan")
```

Comment:
```suggestion
@Deprecated("Use Tracer.trace or Tracer.traceWithSpan instead")
```

If it's deprecated, I think that already implies `Don't create new spans this way`